### PR TITLE
[WIP] initial update of syllabus and schedule

### PR DESCRIPTION
**Pull request by mlp6**
* updated syllabus learning objectives
* updates lecture dates, but content still needs to be populated with tasks more specific to new project @suyashkumar I'll let you work backwards from the end of the semester for what you think is needed then since you'll own most of those lectures, and I'll fill in from the front of the semester to the point where you'll take over (letting you guide that)
* consider letting the students set their MVP targets at some point?  prescribe them early on?

### CODE REVIEW

File: syllabus.tex [hunk 1]
```diff
@@ -4,51 +4,55 @@
...
     \item Define software specifications and constraints
+    \item Agile project management
+    \begin{itemize}
+        \item Team roles
+        \item User stories
+        \item Backlog, To Do, In Progress, Completed, Blocked ``Issues'' (Kanban)
+        \item Weekly sprints
+        \item MVPs
+    \end{itemize}
     \item Device programming fundamentals
```
Comment (suyashkumar) [inline syllabus.tex]: If you think it's appropriate I can add an `\item` for something like Server side (or cloud based) software design and include items like 
* HIPPA Compliance in the cloud
* RESTful API Design
* Scalable compute infrastructure in the cloud via Remote Procedure Calls (RPCs)
* HTTP 
* Python server side frameworks (Flask)

File: syllabus.tex [hunk 2]
```diff
@@ -4,51 +4,55 @@
...
-        \item Convolution \& correlation
-        \item Filtering
-        \item Peak detection
-        \item Envelope detection
-        \item Spectral analysis
-        \item Wavelet analysis
-    \end{itemize}
+    \item Software version control (git, GitHub)
     \item Documentation
     \begin{itemize}
```
Comment (suyashkumar) [inline syllabus.tex]: should we mention the software design doc here?

File: syllabus.tex (hunk 1 above)
Comment (mlp6) [reply syllabus.tex]: Please do...

**Comment by mlp6** [comment]
@suyashkumar I'll check back this evening to see if you push up your additions

File: syllabus.tex [hunk 3]
```diff
@@ -4,51 +4,68 @@
...
-    \item Software version control (git, GitLab)
-    \item Biosignals
-    \begin{itemize}
-        \item Anatomy/physiology review
-        \item Signal transduction
-        \item Noise \& artifacts
-    \end{itemize}
-    \item Signal processing
+    \item Backend Software Development in the Cloud
     \begin{itemize}
```
Comment (suyashkumar) [inline syllabus.tex]: @mlp6 Put some early ideas down here--let me know if they're too general or if they don't fit, and can modify

Comment (suyashkumar) [reply syllabus.tex]: might add a [TBD] for a databases intro

**Pull request merged**